### PR TITLE
adding js for fix

### PR DESCRIPTION
**Pull request by FinalAngel**
includes debugging consoles for now

### CODE REVIEW

File: cms/static/cms/js/modules/cms.plugins.js [hunk 1]
```diff
@@ -778,6 +802,16 @@ var CMS = window.CMS || {};
             },
 
             /**
+             * Changes the settings attributes on an initialised plugin
+             *
+             * @method _setSettings
+             * @private
+             */
+            _setSettings: function _setSettings() {
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.plugins.js]: ymmv

```
_setSettings(newSettings) {
      var settings  = $.extend(true, {}, old, newSettings)
     this.plugin.data('settings', settings)
}

_setSettings({ urls: response.urls })
```

Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.plugins.js]: :+1: will do that once we clarify the outstanding issues with @czpython

**Comment by czpython** [comment]
@FinalAngel Updated my branch

**Comment by vxsx** [comment]
:+1:

**Pull request merged**